cEP-0014.md

```diff
@@ -351,3 +351,409 @@ A possible set of deault capabilties can be:
 4. The filtering and selection of bears take place as described in sections above.
 5. After the bears are selected for every language present in the project, coala-quickstart tries automatically to fill the setting values for the bears, failing which, the bear is discarded. In case some inconsistency is encountered here, an attempt is made to resolve it based on scope, failing which, the bear may be discarded, or any one of the possible value is filled in with a warning comment.
 6. The per-language sections are generated, and final `.coafile` is written.
+
+
+## Code Samples
+
+### Collecting extracted information from project
+```py
+from coala_quickstart.info_extractors.GruntfileInfoExtractor import (
+    GruntfileInfoExtractor)
+from coala_quickstart.info_extractors.EditorconfigInfoExtractor import (
+    EditorconfigInfoExtractor)
+from coala_quickstart.info_extractors.PackageJSONInfoExtractor import (
+    PackageJSONInfoExtractor)
+from coala_quickstart.info_extractors.PackageJSONInfoExtractor import (
+    GemfileInfoExtractor)
+
+
+def collect_info(project_dir):
+    gruntfile_info = GruntfileInfoExtractor(
+        ["Gruntfile.js"], project_dir).extract_information()
+
+    editorconfig_info = EditorconfigInfoExtractor(
+        [".editorconfig"], project_dir).extract_information()
+
+    package_json_info = PackageJSONInfoExtractor(
+        ["package.json"], project_dir).extract_information()
+
+    gemfile_info = GemfileInfoExtractor(
+        ["Gemfile"], project_dir).extract_information()
+
+    extracted_info = aggregate_info(
+        gruntfile_info, editorconfig_info, package_json_info, gemfile_info)
+
+    return extracted_info
+
+
+def aggregate_info(*args):
```

Comment:
I'd rather this accept an array of `InfoExtractor` instances instead of `*args`. It's more intuitive for this use-case IMO.